### ISSUE TITLE
Can't install on Statamic ^3.3

### ISSUE THREAD

**Issue by justinhough** [jonassiewertsen/statamic-oh-dear#7]
Was using this plugin in Statamic version 3.3 and could not install.

Here is the error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires jonassiewertsen/statamic-oh-dear ^1.6 -> satisfiable by jonassiewertsen/statamic-oh-dear[v1.6.0].
    - jonassiewertsen/statamic-oh-dear v1.6.0 requires statamic/cms 3.0.* || 3.1.* || 3.2.* -> found statamic/cms[v3.0.0-beta.1, ..., 3.2.x-dev] but it conflicts with your root composer.json require (3.3.8).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by justinhough** [comment]
@jonassiewertsen I put in a PR for the change needed.

**Comment by jonassiewertsen** [comment]
Nice to see that this addon is still beeing used. 

 @justinhough I can see the branch with the fix, but the PR itself is missing. Can you add it?

**Comment by justinhough** [comment]
@jonassiewertsen Sorry. Just added it.

**Comment by jonassiewertsen** [comment]
Thanks

**Comment by jonassiewertsen** [comment]
Closed via #8
